wiki/Tournaments/OMIC/2019_4K/en.md

```diff
@@ -2,9 +2,448 @@
+| [Slowless](https://osu.ppy.sh/users/10053987) ::{ flag=ID }:: | 0 | **4** | ::{ flag=ID }:: **[Heriza-](https://osu.ppy.sh/users/8911771)** | [#1](https://osu.ppy.sh/community/matches/53307906) |
+| **[reyss](https://osu.ppy.sh/users/4557440)** ::{ flag=ID }:: | **4** | 0 | ::{ flag=ID }:: [Onii\_ue](https://osu.ppy.sh/users/12471868) | -win by default- |
+| [Neptunia-chan](https://osu.ppy.sh/users/11608122) ::{ flag=ID }:: | 0 | **4** | ::{ flag=ID }:: **[ChocoCoki-](https://osu.ppy.sh/users/3565870)** | -win by default- |
+| **[\_claidheamh](https://osu.ppy.sh/users/4686520)** ::{ flag=ID }:: | **4** | 0 | ::{ flag=ID }:: [StyxFearNite](https://osu.ppy.sh/users/8390933) | -win by default- |
+
+Sunday, 14 July 2019
+
+| Player 1 | | | Player 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| [Sadatsuka-Chan](https://osu.ppy.sh/users/9670957) ::{ flag=ID }:: | 0 | **4** | ::{ flag=ID }:: **[alvinrainbow](https://osu.ppy.sh/users/4783042)** | -win by default- |
+| [Sadatsuka-Chan](https://osu.ppy.sh/users/9670957) ::{ flag=ID }:: | 0 | **4** | ::{ flag=ID }:: **[Artem1s](https://osu.ppy.sh/users/12561379)** | -win by default- |
+| **[Yamada Kitaro](https://osu.ppy.sh/users/7670805)** ::{ flag=ID }:: | **4** | 2 | ::{ flag=ID }:: [\[KonG-69\]](https://osu.ppy.sh/users/8847412) | [#1](https://osu.ppy.sh/community/matches/53328132) |
+| [\[ID\]Catalyst-](https://osu.ppy.sh/users/8810018) ::{ flag=ID }:: | 0 | **4** | ::{ flag=ID }:: **[Yamada Kitaro](https://osu.ppy.sh/users/7670805)** | -win by default- |
+| **[Yangga](https://osu.ppy.sh/users/12143152)** ::{ flag=ID }:: | **4** | 0 | ::{ flag=ID }:: [Its A Name](https://osu.ppy.sh/users/8724363) | [#1](https://osu.ppy.sh/community/matches/53328679) |
+| **[\[ID\]Catalyst-](https://osu.ppy.sh/users/8810018)** ::{ flag=ID }:: | **4** | 3 | ::{ flag=ID }:: [rizki20](https://osu.ppy.sh/users/6523286) | [#1](https://osu.ppy.sh/community/matches/53329314) |
+| **[Artem1s](https://osu.ppy.sh/users/12561379)** ::{ flag=ID }:: | **4** | 0 | ::{ flag=ID }:: [Zero\[13\]](https://osu.ppy.sh/users/13942350) | [#1](https://osu.ppy.sh/community/matches/53331799) |
+| [Yangga](https://osu.ppy.sh/users/12143152) ::{ flag=ID }:: | 0 | **4** | ::{ flag=ID }:: **[-\[ChoKy\]-](https://osu.ppy.sh/users/4691146)** | [#1](https://osu.ppy.sh/community/matches/53331589) |
+| **[Slowless](https://osu.ppy.sh/users/10053987)** ::{ flag=ID }:: | **4** | 0 | ::{ flag=ID }:: [MATEH ON](https://osu.ppy.sh/users/8136810) | [#1](https://osu.ppy.sh/community/matches/53333251) |
+| **[-\[ChoKy\]-](https://osu.ppy.sh/users/4691146)** ::{ flag=ID }:: | **4** | 0 | ::{ flag=ID }:: [\[-TiaZark-\]](https://osu.ppy.sh/users/14294140) | [#1](https://osu.ppy.sh/community/matches/53333168) |
+
+### Round of 56
+
+Saturday, 29 June 2019
+
+| Group | Player 1  | Player 2 | Player 3 | Player 4 | Player 5 | Match Link |
```

Comment:
can't this be just `| Group | Players | Match link |`?

Reply:
sure